src/Resources/views/CRUD/Association/edit_many_script.html.twig

```diff
@@ -462,6 +462,19 @@ This code manages the many-to-[one|many] association field popup
 
             Admin.log('[{{ id }}|on:change] update the label');
 
+            var objectId =  jQuery(this).val();
+
+            // Skip short object information call if object is missing.
+            if (!objectId) {
```

Comment:
Even if is border case, values like `0` are valid for ids.
IMO, we should make this comparison strict.

Reply:
What do you think if we make the comparison like this one:
`if (objectId === undefined || objectId === null || objectId === "") {`

Reply:
It would work but I don't know if all the value are possible. Maybe someone knows.